pwa/src/components/attributes/attribuutForm.tsx

```diff
@@ -87,18 +88,18 @@ export default function AttributeForm({ id, entity }) {
     let allOf = retrieveFormArrayAsOArray(event.target, "allOf");
     let anyOf = retrieveFormArrayAsOArray(event.target, "anyOf");
     let oneOf = retrieveFormArrayAsOArray(event.target, "oneOf");
-    let forbiddenIf = retrieveFormArrayAsOArray(event.target, "forbidenIf");
+    let forbidenIf = retrieveFormArrayAsOArray(event.target, "forbidenIf");
```

Comment:
De juiste spelling zou zijn `forbiddenIf` (kan dus hier en overal door dit bestand heen geüpdatet worden)

Reply:
Ik heb een spelfout gecorrigeerd maar kijken naar  deze spelfout ziet ook in datatable.